com.unity.ml-agents/Runtime/Sensors/VectorSensorComponent.cs

```diff
@@ -0,0 +1,56 @@
+using UnityEngine;
+using UnityEngine.Serialization;
+
+namespace Unity.MLAgents.Sensors
+{
+    [AddComponentMenu("ML Agents/Vector Sensor", (int)MenuGroup.Sensors)]
+    public class VectorSensorComponent : SensorComponent
```

Comment:
Please add docstrings here and on public properties.

Reply:
Ok, added the docstring here and on the properties.